nms/app/packages/magmalte/app/views/subscriber/SubscriberTable.js

```diff
@@ -110,6 +118,7 @@ function SubscribersTable(props: WithAlert) {
   return (
     <>
       <div className={classes.dashboardRoot}>
+        <CardTitleRow key="title" icon={SettingsIcon} label={'Subscribers'} />
         <JsonDialog open={jsonDialog} onClose={onClose} imsi={currRow.imsi} />
         <ActionTable
           tableRef={tableRef}
```

Comment:
In action table, can you add a field `localization` https://github.com/magma/magma/blob/4a92d2280b005d35cc95d7a6896dc863b2fa922c/nms/app/packages/magmalte/app/components/DashboardAlertTable.js#L297 
As shown above. We can set the search string hint through this. We can set it to provide information on what user should search for. For e.g IMSI010101010101